### PR TITLE
Feature: Recommended Unity .gitignore (#5)

### LINKED ISSUE #5: Update gitignore with Unity gitignore
We should update our .gitignore so that we have the recommended Unity gitignore settings for GitHub.

### PR DESCRIPTION
**Pull request by AndreasHeiskanen**
Update .gitignore with recommended Unity .gitignore for GitHub repositories with multiple contributors. Fixes #5.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,33 +1,59 @@
...
-# OS generated #
-# ============ #
-.DS_Store
-.DS_Store?
-._*
-.Spotlight-V100
-.Trashes
-Icon?
-ehthumbs.db
-Thumbs.db
```
Comment (joribom) [inline .gitignore:33]: These lines don't seem to be in the new one, should we not include them as well?

File: .gitignore [hunk 2]
```diff
@@ -1,33 +1,59 @@
-# =============== #
-# Unity generated #
-# =============== #
-Temp/
-Obj/
-UnityGenerated/
```
Comment (joribom) [inline .gitignore:6]: UnityGenerated is not in the new file, should it not be ignored?

File: .gitignore (hunk 1 above)
Comment (AndreasHeiskanen) [reply .gitignore:33]: These ones are actually not needed in the .gitignore for our project.

File: .gitignore (hunk 2 above)
Comment (AndreasHeiskanen) [reply .gitignore:6]: Temp/ is replaced with lower case temp/
Obj/ is replaced with lower case obj/
UnityGenerated is not something that we need in our project and can be removed.

**Review by joribom — APPROVED** [review]

**Pull request merged**